optaplanner-core/src/test/java/org/optaplanner/core/impl/localsearch/decider/forager/finalist/StrategicOscillationByLevelFinalistPodiumTest.java

```diff
@@ -212,4 +213,8 @@ protected LocalSearchMoveScope buildMoveScope(LocalSearchStepScope stepScope,
         return moveScope;
     }
 
+    private static <E> void verifyListContainsExactlyOneInstance(List<E> list, E instance) {
```

Comment:
@jhrcek Yes, you are almost correct. It serves the same purpose but there is a slight difference. The `verifyListContainsExactlyOneInstance` method makes sure that the single item in the list is exactly the expected _instance_. `containsExactly()`, on the other hand, only verifies the list items are _equal_ to the expected ones.

I was very careful to make _equivalent_ changes during the migration, otherwise I would use `containsExactly()`.